text/0685-new-browser-support-policy.md

```diff
@@ -0,0 +1,293 @@
+   whose version is tied to the version of macOS and iOS that users use. As
+   users will often wait to update their operating system, many users lag behind
+   on the version of Safari they are using, so it cannot be considered
+   evergreen.
+
+Using these categories, Ember will support the following browsers:
+
+- Evergreen
+  - Desktop
+      1. Google Chrome/Chromium
+      2. Mozilla Firefox
+      3. Microsoft Edge
+  - Mobile
+      1. Google Chrome
+      2. Mozilla Firefox
+
+- Non-evergreen
+  - Desktop
+    1. Safari
+  - Mobile
+    1. Safari
+
+Any browser which is not listed here may work, but is not explicitly supported.
+In addition, in order to recategorize browsers as evergreen or non-evergreen, a
+new RFC must be submitted to update the support policy.
+
+So for instance, if Safari were to begin automatically updating its version
+independently of macOS/iOS versions, and thus fulfill the criteria for evergreen
+browsers, it would still be treated as a non-evergreen browser until this policy
+is updated. This will prevent sudden and unexpected changes in the support
+matrix for Ember users.
+
+### Evergreen browsers
+
+Ember and Ember Data will support **all** versions of evergreen browsers which
+meet any of the following criteria:
+
+1. It is the latest version of the browser.
+2. It is the latest LTS/extended support version of the browser (such as Firefox ESR).
+3. It has at least **0.25%** marketshare usage, based on
+   [statcounter](https://gs.statcounter.com/).
+
+This policy will allow Ember to generally support the most recent 2-3 versions
+of the browser, and account for versions which may become temporary transition
+points. For example, when MS Edge made the switch to the new Chromium-backed
+version, the last non-Chromium version saw higher usage for a slightly longer
+period of time as the update process was not as standard, and required an
+explicit opt-in. These transitionary periods are important, but generally
+speaking temporary, and once the usage drops below the threshold Ember can drop
+support at that point.
+
+The supported versions of browsers are determined at the time of each _minor_
+release of Ember and Ember Data. In the release blog post, we will specify
+which versions of browsers are supported for this minor version, to keep users
+informed as usage changes.
+
+_Within_ a minor version, support works as follows:
+
+1. Support for a given browser version is _never_ removed.
+2. Support is _added_ for any new versions of a browser that have been released
+   while the minor is still supported.
+
+So for instance, if we were to release Ember 4.4 with support for Chrome 92, 93,
+and 94, then bugfixes and security fixes for those versions of Chrome will be
+supported until 4.4 is no longer an LTS, following the LTS support matrix. In
+addition, if Chrome were to release version 95 while 4.4 is still actively
+supported, then 4.4 will also support Chrome 95.
```

Comment:
How do you know a version is dropped? What does a user actually have to do to know a specfic chrome version they care about is no longer supported?

Reply:
Every minor release of Ember will contain the list of minimum supported versions in the blog post. I'm going to expand the "how we teach this section" to include this.